### ISSUE TITLE
Error on https://f.silvergames.com/ruffle/player.php

### ISSUE THREAD

**Issue by nguie2** [ruffle-rs/ruffle#10699]
# Error Info
Error name: Error
Error message: panicked at 'called `Option::unwrap()` on a `None` value', core/src/bitmap/bitmap_data.rs:595:51
Error stack:
```
Error: panicked at 'called `Option::unwrap()` on a `None` value', core/src/bitmap/bitmap_data.rs:595:51
    at O.n.wbg.__wbg_new_15d3966e9981a196 (https://f.silvergames.com/ruffle/23042023/core.ruffle.83032e5cb08379aef3d6.js:1:47895)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::h927501cce3fc7417 (wasm://wasm/02acc5e6:wasm-function[8165]:0x8010b2)
    at std::panicking::rust_panic_with_hook::hc53aea0352e77326 (wasm://wasm/02acc5e6:wasm-function[5438]:0x735478)
    at core::panicking::panic_fmt::hf4a9df75710ece83 (wasm://wasm/02acc5e6:wasm-function[8047]:0x7fbd21)
    at core::panicking::panic::h54beb3f515dd3c6d (wasm://wasm/02acc5e6:wasm-function[7453]:0x7c8944)
    at ruffle_core::bitmap::bitmap_data::BitmapData::update_dirty_texture::hee3ddbdfbdc50e9d (wasm://wasm/02acc5e6:wasm-function[1254]:0x3bec50)
    at <ruffle_core::library::MovieLibrarySource as ruffle_render::bitmap::BitmapSource>::bitmap_handle::h0909caf8b3654249 (wasm://wasm/02acc5e6:wasm-function[3183]:0x6168dc)
    at <ruffle_render_wgpu::backend::WgpuRenderBackend<T> as ruffle_render::backend::RenderBackend>::register_shape::hd515b52f2b24d068 (wasm://wasm/02acc5e6:wasm-function[1040]:0x3550d5)
    at ruffle_core::display_object::movie_clip::MovieClipData::define_shape::h2a01979e41768b81 (wasm://wasm/02acc5e6:wasm-function[2992]:0x5f64b2)
    at ruffle_core::display_object::movie_clip::MovieClip::preload::h8b435e95ba744ef1 (wasm://wasm/02acc5e6:wasm-function[617]:0x1d75a3)
```

# Player Info
Allows script access: false

# Page Info
Page URL: https://f.silvergames.com/ruffle/player.php?id=273
SWF URL: https://f.silvergames.com/TU-95.swf

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-04-02
Channel: nightly
Built: 2023-04-02T00:19:29.297Z
Commit: 58cbc38983ff175005522d009da3fbfcf6208ef3
Is extension: false

# Metadata
width: 670
height: 450
frameRate: 24
numFrames: 31
swfVersion: 9
backgroundColor: #000000
isActionScript3: false
uncompressedLength: 3871229

**Comment by n0samu** [comment]
Yet another regression caused by #10315 :/

**Comment by n0samu** [comment]
Probably the same regression as #10666

**Comment by Lord-McSweeney** [comment]
Fixed by #10908.
